src/overlays/actors/ovl_Bg_Toki_Swd/z_bg_toki_swd_cutscene_data.c

```diff
@@ -0,0 +1,274 @@
+#include "z_bg_toki_swd.h"
+
+// clang-format off
```

Comment:
I think it's really unfortunate that we have to disable clang-format for every single one of those cutscene data sections. We might want to think of a different solution for lists that doesn't require indenting, or one that would work fine with auto formatting.